api/src/main/java/jakarta/data/metamodel/impl/ComparableAttributeRecord.java

```diff
@@ -1,34 +0,0 @@
-/*
- * Copyright (c) 2025 Contributors to the Eclipse Foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-package jakarta.data.metamodel.impl;
-
-import jakarta.data.Sort;
-import jakarta.data.metamodel.ComparableAttribute;
-
-public record ComparableAttributeRecord<T,V extends Comparable<V>>(String name)
-        implements ComparableAttribute<T,V> {
-    @Override
-    public Sort<T> asc() {
-        return Sort.asc(name);
-    }
-
-    @Override
-    public Sort<T> desc() {
-        return Sort.desc(name);
-    }
-}
```

Comment:
So, given how trivial these are (they're not a burden in terms of maintenance), I would not delete these unless we're also deprecating the existing implementations in Jakarta Data 1.1. I think it's completely fine to have that conversation, but it seems out of scope for this pull request.

Reply:
Sure, I'll add them back in and we can split that discussion out separately.

Reply:
Commit https://github.com/jakartaee/data/pull/976/commits/0427cdd04f753336683a8e5e7de40984a87aa65c is ready for review. It adds the implementation classes back in, fixes two typos, adds the import for usage of java.util.Ojbects, and adds a clarification to account for primitives like `int` corresponding to ComparableAttribute on the basis of the wrapper class, even though the primitive itself isn't Comparable